### PR TITLE
complete the task

### PR DESCRIPTION
**Pull request by barsuchellos**
[DEMO LINK](https://barsuchellos.github.io/react_person/)

https://prnt.sc/Wdp-E8BT-bUe

### CODE REVIEW

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,33 @@
...
+        (!isMarried)
+          ? (
+            <p className="Person__partner">
+              I am not married
+            </p>
+          )
+          : (
+            <p className="Person__partner">
+              {`${partnerName} is my ${partner}`}
+            </p>
```
Comment (xiimiik) [inline src/components/Person/Person.jsx:28]: paragraph and class dont change, so change only the filling

**Review by xiimiik — CHANGES_REQUESTED** [review]
GJ, but need some fix

**Review by xiimiik — APPROVED** [review]
Nice!

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,23 @@
...
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+      )}
```
Comment (Anna-Tkachenko) [inline src/components/Person/Person.jsx:16]: fix indents
```suggestion
      {age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
```

File: src/components/Person/Person.jsx [hunk 3]
```diff
@@ -1 +1,23 @@
...
+
+      <p className="Person__partner">
+        {!isMarried ? 'I am not married' : `${partnerName} is my ${partner}`}
```
Comment (Anna-Tkachenko) [inline src/components/Person/Person.jsx:19]:
```suggestion
        {!isMarried
          ? 'I am not married'
          : `${partnerName} is my ${partner}`}
```

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

**Review by Anna-Tkachenko — APPROVED** [review]

**Pull request closed without merging**